hazelcast/src/main/java/com/hazelcast/security/permission/ActionConstants.java

```diff
@@ -65,6 +66,13 @@
     public static final String ACTION_PROJECTION = "projection";
     public static final String ACTION_USER_CODE_DEPLOY = "deploy";
 
+    public static final String ACTION_LIST = "list";
+    public static final String ACTION_SUSPEND = "suspend";
+    public static final String ACTION_RESUME = "resume";
+    public static final String ACTION_EXPORT = "export";
+    public static final String ACTION_UPLOAD = "upload";
```

Comment:
This action is not implemented.

Reply:
this is used on the EE side to grant permission to create/put actions for jet internal maps.